Cargo.toml

```diff
@@ -146,6 +146,10 @@ incremental = true
 lto = true
 opt-level = 'z' # Optimize for size
 
+[profile.release-with-debug]
+inherits = "release"
+debug = true
```

Comment:
I added this because I thought it's helpful for more easily doing release build with debug symbols (`cargo build --profile=release-with-debug`)

Reply:
Interesting, can you add a comment about it here?

Reply:
Yup, I have a comment in my local already :) Just waiting for other feedback before doing a CI run.